.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint',
+  ],
+  rules: {
+    "@typescript-eslint/no-unused-vars": [2]
```

Comment:
Nit - can we use "error" instead of 2? Otherwise one has to understand ESLint's configuration to understand what a '2' is.
```suggestion
    "@typescript-eslint/no-unused-vars": ["error"]
```